lib/shopwareStoreCommander.js

```diff
@@ -63,16 +63,20 @@ module.exports = class ShopwareStoreCommander {
     /**
      * @param {String} pluginName
      * @param {Object} extraFields - optional
-     * @return {Promise}
+     * @return {Object}
      */
     async findPlugin(pluginName, extraFields) {
-        const accountData = await ((!this.accountData.plugins)
-            ? this.loadAccountData()
-            : (Promise.resolve(this.accountData)));
+        let accountData = null;
+        if (!this.accountData.plugins) {
+            accountData = await this.loadAccountData();
+        } else {
+            accountData = this.accountData;
```

Comment:
The `else`-clause is kind of redundant, you can just use this as the initial value above and then remove the `this` part in the `if` condition above. Also, having `loadAccountData` both return the account data store it on the instance is somewhat confusing, you should do either one or the other, but not both.